Options.py

```diff
@@ -902,13 +902,19 @@ def __contains__(self, item):
 class OptionSet(Option[typing.Set[str]], VerifyKeys):
     default = frozenset()
     supports_weighting = False
+    random_str = None
 
-    def __init__(self, value: typing.Iterable[str]):
+    def __init__(self, value: typing.Iterable[str], random_str: str = None):
```

Comment:
`None` is not a string

Reply:
For more context, pep 484 says:
> A past version of this PEP allowed type checkers to assume an optional type when the default value is None, as in this code:
> 
> `def handle_employee(e: Employee = None): ...`
> 
> This would have been treated as equivalent to:
> 
> `def handle_employee(e: Optional[Employee] = None) -> None: ...`
> 
> This is no longer the recommended behavior. Type checkers should move towards requiring the optional type to be made explicit.
```suggestion
    def __init__(self, value: typing.Iterable[str], random_str: str | None = None):
```